pages/_app.js

```diff
@@ -1,7 +1,6 @@
-import '../styles/globals.css'
+import '../styles/Globals.css'
 
-function MyApp({ Component, pageProps }) {
+// This default export is required in a new `pages/_app.js` file.
+export default function MyApp({ Component, pageProps }) {
```

Comment:
You do `export default function MyApp` here and in other files you do `export default Birthdays;`
Choose one and stick to it.